src/utils/futil.js

```diff
@@ -12,3 +12,6 @@ export let FlattenTreeLeaves = Tree =>
   )
 export let PlainObjectTree = F.tree(onlyWhen(_.isPlainObject))
 export let flattenPlainObject = F.whenExists(FlattenTreeLeaves(PlainObjectTree))
+
+// Array
+export let len = _.get('length')
```

Comment:
This is very close to `lens` which would be confusing if it was in the futil API. Ramda has a `length` function so there's precedent there if we really want to add this.

Reply:
Yes. Just let me know if I should put it somewhere else.